### DISCUSSION TITLE
Training a detection model on custom data

### DISCUSSION THREAD

**Discussion by kulsoom-abdullah** [JaidedAI/EasyOCR#927] [Q&A]
Hello,
I have found and gone over the tutorials for training a custom recognition model using generated or existing data. For example: https://github.com/JaidedAI/EasyOCR/issues/495 

Any guidance out there for training the detector part on existing data? 

longer explanation:
I used code from [keras_ocr](https://github.com/faustomorales/keras-ocr) to use existing background images I have and generate synthetic text embedded onto these background images.  The output is the background image with the text on top along with the corresponding ground truth - bounding box coordinates for each character. 

I would like to use this to fine tune the easy ocr detector.  

Thanks in advance!

Update: I found the following on training the CRAFT detection model https://github.com/JaidedAI/EasyOCR/blob/master/trainer/craft/README.md

**Comment by masoudMZB** [comment]
@kulsoom-abdullah did finetune detection model? I face some problem while finetuning. this is my issue https://github.com/JaidedAI/EasyOCR/issues/1005

**Reply by kulsoom-abdullah** [reply]
I fortunately figured out how to fine tune the detection model with my own images and labels using the craft trainer. 
What is your status now?

**Reply by masoudMZB** [reply]
no progress since then,

**Comment by tam0w** [comment]
@kulsoom-abdullah did you train the new craft model with all fresh images? or did you use some pictures that were originally used to train the craft model? can you help me: id like to fine tune the craft model to detect single numeric character per line in images since that isnt being detected right now
